src/Controls/tests/TestCases.HostApp/Issues/Issue20842.xaml.cs

```diff
@@ -13,8 +13,8 @@ public override string ToString()
 
 	public class PersonDataTemplateSelector : DataTemplateSelector
 	{
-		public required DataTemplate ValidTemplate { get; set; }
-		public required DataTemplate InvalidTemplate { get; set; }
+		public DataTemplate ValidTemplate { get; set; }
+		public DataTemplate InvalidTemplate { get; set; }
```

Comment:
Just checking why this was made? Was it invalid before or is there a change in XSG to make it work coming later?